angular/projects/catalyst/src/lib/dialog/dialog.component.ts

```diff
@@ -16,6 +25,26 @@ import { CatDialogHeaderComponent } from './dialog-header.component';
   }
 })
 export class CatDialogComponent implements AfterContentInit {
+  /**
+   * Flag to disable/enable overscroll behavior.
+   */
+  @Input() noOverscroll = false;
+
+  /**
+   * Flag to not fire an initial scroll event after content initialization.
+   */
+  @Input() noContentScrolledInit = false;
```

Comment:
What is your reasoning for renaming the inputs?

Reply:
Yea, probably there is no point to rename it, can't see a real scenario that we might have a scrollable header or actions. I'll rename it back